std_msgs/Header header
string[] defender_cmd
string[] attacker_cmd
bool allCaptured
bool enemy_win
bool gameEnd
